demo/demo.js

```diff
@@ -6,23 +6,22 @@ function parseRecord(record) {
   if (!nonvisualVariants[0]) throw new Error('No nonvisual variants');
 
   return {
-    data,
-    firstVariant,
-    nonvisualVariants,
-    visualVariants,
+    data: data,
+    firstVariant: firstVariant,
+    nonvisualVariants: nonvisualVariants,
+    visualVariants: visualVariants,
   };
 }
 
 function parseVariants(variants) {
-  return variants.map((v) => ({
-    color: v.standardColor,
-    text: v.standardColor,
-    image: v.swatchImage,
-    product: {
-      imageSrc: v.productImage,
-      imageAlt: v.description,
-    },
-  }));
+  return variants.map(function(v) {
+    return {
+      color: v.standardColor,
+      text: v.standardColor,
+      image: v.swatchImage,
+      product: {imageSrc: v.productImage, imageAlt: v.description, },
```

Comment:
Jus a space missing here:
```suggestion
      product: { imageSrc: v.productImage, imageAlt: v.description, },
```